test/categorical_algebra/CSets.jl

```diff
@@ -309,6 +309,20 @@ pb = ob(pullback(ϕ, ψ))
 @test (nv(pb), ne(pb)) == (8, 2)
 @test Set(pb[:weight]) == Set([(+1, -1), (-1, +1)])
 
+# Limit with LooseACSetTransformations 
+A = @acset VELabeledGraph{Symbol} begin V=2; vlabel=[:a,:b] end;
+B = @acset VELabeledGraph{Symbol} begin V=2; vlabel=[:q,:z] end;
+C = @acset VELabeledGraph{Symbol} begin V=2; vlabel=[:x,:y] end;
+ac = LooseACSetTransformation(
+  Dict([:V=>[1,2]]),Dict([:Label=>FinFunction(Dict(:a=>:x,:b=>:y))]),A,C);
```

Comment:
Not important, but you can save a few characters by replacing `Dict([:V=>[1,2]])` with `Dict(:V=>[1,2])`, etc.